.github/workflows/python-package.yml

```diff
@@ -41,3 +43,8 @@ jobs:
       run: |
         if python --version 2>&1 | grep -q "Python 2" || python --version 2>&1 | grep -q "Python 3.5" || python --version 2>&1 | grep -q "Python 3.6" ; then for synctest in $(cd tests && ls test*.py | grep -v async); do python -m unittest discover -s tests/ -t . -p "$synctest" || exit 1; done; fi
         if python --version 2>&1 | grep -q "Python 3.7" || python --version 2>&1 | grep -q "Python 3.8" || python --version 2>&1 | grep -q "Python 3.9" || python --version 2>&1 | grep -q "Python 3.10"; then coverage run --source adb_shell -m unittest discover -s tests/ -t . && coverage report -m && coveralls; fi
+    - name: Upload wheel as a workflow artifact
```

Comment:
I haven't used GitHub Actions to upload an artifact. What's the benefit of doing so?

Reply:
> What's the benefit of doing so?

The ability to download created files (e.g. wheels) to use them somehow. For example, to get a proper wheel before making a release.